src/routes/committee/schedule/[rehearsalId]/+page.ts

```diff
@@ -44,28 +44,19 @@ export const load: PageLoad = async ({ params }) => {
 		const songIds = rehearsal.songsToRehearse.map((rehearsalSong) => rehearsalSong.song.id)
 
 		// Get the song objects from the rehearsal songs
-		const songsQuery = query(collection(db, 'songs'), where('__name__', 'in', songIds))
-		const songDocs = (await getDocs(songsQuery)).docs
+		const songDocs = await QueryWhereIn(collection(db, 'songs'), '__name__', songIds)
 		songs = songDocs.map((doc) => ({ id: doc.id, ...doc.data() } as song))
 
-		// Get the playsSongInEdition for all participants on this rehearsal day
-		// const playsInQuery = query(
-		// 	collectionGroup(db, 'playsSongInEdition'),
-		// 	where(
-		// 		'song',
-		// 		'in',
-		// 		songDocs.map((song) => song.ref)
-		// 	)
-		// )
-		// const playsInDocs = (await getDocs(playsInQuery)).docs
-
-		const playsInQuery = query(collectionGroup(db, 'playsSongInEdition'))
-		const playsInDocs = (await getDocs(playsInQuery)).docs
+		const playsInDocs = await QueryWhereIn(
+			collectionGroup(db, 'playsSongInEdition'),
+			'song',
+			songDocs.map((song) => song.ref)
+		)
 
 		// Get all the participants that play these songs
 		const participantIds = playsInDocs.map((doc) => doc.ref.parent.parent?.id as string)
-		const participantQuery = query(collection(db, 'users')) //, where('__name__', 'in', participantIds))
-		const participants = (await getDocs(participantQuery)).docs.map(
+		const participantDocs = await QueryWhereIn(collection(db, 'users'), '__name__', participantIds)
+		const participants = participantDocs.map(
 			(participant) =>
```

Comment:
Misschien hier ff een todo bij zetten om later gewoon alle participants van de huidige editie te doen? Waarschijnlijk speelt toch bijna iedereen en is dat efficienter